fe/fe-core/src/main/java/com/starrocks/sql/plan/PlanFragmentBuilder.java

```diff
@@ -1205,18 +1205,25 @@ public PlanFragment visitPhysicalHashAggregate(OptExpression optExpr, ExecPlan c
             TupleDescriptor outputTupleDesc = context.getDescTbl().createTupleDescriptor();
 
             ArrayList<Expr> groupingExpressions = Lists.newArrayList();
+            // EXCHANGE_BYTES/_SPEED aggregate the total bytes/ratio on a node, without grouping, remove group-by here.
+            // the group-by expressions just denote the hash distribution of an exchange operator.
+            boolean forExchange = node.getAggregations().values().stream().anyMatch(aggFunc ->
```

Comment:
would better give a better name to `forExchange`

Reply:
changed to forExchangePerf